tests/baselines/reference/undefinedTypeAssignment2.errors.txt

```diff
@@ -1,8 +1,8 @@
-undefinedTypeAssignment2.ts(1,5): error TS2397: Declaration name conflicts with built-in global identifier 'undefined'.
+undefinedTypeAssignment2.ts(1,5): error TS2300: Duplicate identifier 'undefined'.
 
 
 ==== undefinedTypeAssignment2.ts (1 errors) ====
     var undefined = void 0;
         ~~~~~~~~~
-!!! error TS2397: Declaration name conflicts with built-in global identifier 'undefined'.
+!!! error TS2300: Duplicate identifier 'undefined'.

```

Comment:
This is probably not a great error now that I dropped the builtinGlobals method of doing this. Probably I should keep that code around but only error in value space.